CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2.15.3 (Feb, 4, 2022)
+
+IMPROVEMENTS:
+
+* Added support for manager roles in artifactory_group resource [GH-308]
```

Comment:
Please include the resource name per guideline, see 2.15.2 for example.

Reply:
done